test/components/table/list-view.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+import test from 'ava';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import {shallow, configure} from 'enzyme';
+import {listGranules} from '../../../app/scripts/actions';
+import List from '../../../app/scripts/components/table/list-view';
+import Timer from '../../../app/scripts/components/app/timer.js';
+import {
+  errorTableHeader,
+  errorTableRow, errorTableSortProps
+} from '../../../app/scripts/utils/table-config/granules';
+
+configure({ adapter: new Adapter() });
+
+test('list-view should properly initialize timer config prop', async (t) => {
+  const dispatch = () => Promise.resolve();
+  const query = { q: '_exists_:error AND status:failed' };
+  const list = {
+    meta: {},
+    data: []
+  };
+  const listWrapper = shallow(
+    <List
+      list={list}
+      dispatch={dispatch}
+      action={listGranules}
+      tableHeader={errorTableHeader}
+      sortIdx={4}
+      tableRow={errorTableRow}
+      tableSortProps={errorTableSortProps}
+      query={query}
+    />,
+    {
+      // We must disable lifecycle methods so that when we find the Timer its
+      // properties will still be set to their initial values.  Otherwise, a
+      // lifecycle method might cause the properties to change and we will not
+      // be able to check their initial values, which is what we want to do.
+      disableLifecycleMethods: true,
+      context: {
+        store: {
+          subscribe: () => void 0,
+          dispatch: () => void 0,
+          getState: () => void 0
+        }
+      }
+    }
+  );
+
+  const timerWrapper = listWrapper.dive().find(Timer);
```

Comment:
Interesting that you are doing a find passing in the component itself and not its name. I assume that proved necessary?

Reply:
Not necessary, actually. I could have used the string `'Timer'`, but when I can use specific references rather than strings, I prefer to use them. It enables code analysis to find invalid names prior to runtime. With strings, wrong names aren't found until runtime.